app/views/trainees/lead_schools/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="govuk-heading-l">Lead school</h1>
+
+<%= register_form_with model: @trainee_lead_school_form, url: trainee_lead_schools_path(@trainee), local: true, html: { data: { module: "app-schools-autocomplete" } } do |f| %>
+  <div class="govuk-form-group">
+    <span class="govuk-hint">Search for a school by it's unique reference number (URN), name or postcode</span>
+    <div id="schools-autocomplete-element" class="app-!-autocomplete--max-width-two-thirds"></div>
```

Comment:
It it me or is it full width?

![image](https://user-images.githubusercontent.com/470137/116436062-eef15280-a843-11eb-9c7c-750787859e1c.png)

Reply:
fixed